test_admin_portfolio.py

```
@@ -0,0 +1,55 @@
+"""
+Test if admin user has strategies in portfolio
+"""
+import requests
+import json
+import urllib3
+urllib3.disable_warnings()
+
+BASE_URL = 'https://cryptouniverse.onrender.com/api/v1'
+ADMIN_EMAIL = 'admin@cryptouniverse.com'
+ADMIN_PASSWORD = 'AdminPass123!'
```

Ruff (0.14.1)

Possible hardcoded password assigned to: "ADMIN_PASSWORD"

(S105)